src/main/java/frc/robot/Controls.java

```diff
@@ -496,7 +496,7 @@ private void configureClimbPivotBindings() {
       setClimbLEDs = Commands.none();
     }
 
-    s.climbPivotSubsystem.setDefaultCommand(s.climbPivotSubsystem.advanceClimbCheck());
+    s.climbPivotSubsystem.setDefaultCommand(s.climbPivotSubsystem.advanceClimbCheck().withName("Check Climb"));
```

Comment:
Make the comment "Advance Climb Check", so that the log comment matches the API making it easier to map to.